keras_nlp/models/albert/albert_presets.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""ALBERT model preset configurations."""
+
+
+backbone_presets = {
+    "albert_base_en_uncased": {
+        "config": {
+            "vocabulary_size": 30000,
+            "num_layers": 12,
+            "num_heads": 12,
+            "num_groups": 1,
+            "num_inner_repetitions": 1,
+            "embedding_dim": 128,
+            "hidden_dim": 768,
+            "intermediate_dim": 3072,
+            "dropout": 0.0,
+            "max_sequence_length": 512,
+            "num_segments": 2,
+        },
+        "preprocessor_config": {},
+        "description": (
+            "Base size of ALBERT where all input is lowercased. "
+            "Trained on English Wikipedia + BooksCorpus."
+        ),
+        "weights_url": "https://drive.google.com/uc?export=download&id=1RzTTa8nMcBc84nARvJmHal5SndpKbDUa",
```

Comment:
Still need to review, but this is awesome! Super great to model how to do this for contributors.

Reply:
@mattdangerw, there is a cap on the size of the file you can download from GDrive. For example, `extra_large` and `extra_extra_large` tests fail because their size > 200MB. I have observed the same for FNet. Instead of downloading the actual file, a HTML file is downloaded.